src/uwtools/tests/support.py

```diff
@@ -94,16 +94,18 @@ def regex_logged(caplog: LogCaptureFixture, msg: str) -> bool:
     return any(pattern.search(record.message) for record in caplog.records)
 
 
-def validator(schema_fn: str, *args: Any) -> Callable:
+def schema_validator(schema_name: str, *args: Any) -> Callable:
```

Comment:
Use a more explicit anme to avoid confusion with the name of the `uwtools.config.validator` module.